src/autopas/containers/verletClusterLists/VerletClusterListsRebuilder.h

```diff
@@ -45,37 +46,35 @@ class VerletClusterListsRebuilder {
    * @param clusterList The cluster list to rebuild the neighbor lists for.
    * @param towers The towers from the cluster list to rebuild.
    * @param particlesToAdd New particles to add.
-   * @param newton3 If the neighbor lists should use newton 3.
    */
   VerletClusterListsRebuilder(const VerletClusterLists<Particle> &clusterList,
                               std::vector<ClusterTower<Particle, clusterSize>> &towers,
-                              std::vector<Particle> &particlesToAdd, bool newton3)
+                              std::vector<Particle> &particlesToAdd)
       : _particlesToAdd(particlesToAdd),
         _towers(towers),
         _towerSideLength(clusterList.getTowerSideLength()),
         _interactionLengthInTowers(clusterList.getNumTowersPerInteractionLength()),
         _towerSideLengthReciprocal(1 / _towerSideLength),
         _towersPerDim(clusterList.getTowersPerDimension()),
-        _neighborListIsNewton3(newton3),
         _interactionLength(clusterList.getInteractionLength()),
         _interactionLengthSqr(_interactionLength * _interactionLength),
         _boxMin(clusterList.getBoxMin()),
-        _boxMax(clusterList.getBoxMax()) {}
+        _boxMax(clusterList.getBoxMax()),
+        _haloBoxMin(utils::ArrayMath::subScalar(_boxMin, _interactionLength)),
+        _haloBoxMax(utils::ArrayMath::addScalar(_boxMax, _interactionLength)) {}
```

Comment:
Better and safer to add a getter to `clusterList` instead of doing the calculation twice.